modules/core/src/lib/select-loader.js

```diff
@@ -95,6 +96,16 @@ function findLoaderByExtension(loaders, extension) {
   return null;
 }
 
+// data should be a Response object
+function findLoaderByContentType(loaders, data) {
+  if (!data || !(data instanceof Response)) {
+    return null;
+  }
+
+  const contentType = data.headers.get('content-type');
+  return findLoaderByMimeType(loaders, contentType);
```

Comment:
Can't see the definition of this function.

Reply:
in the original code line 106